src/Compilers/Core/MSBuildTaskTests/VbcTests.cs

```diff
@@ -565,5 +565,39 @@ public void StdLib()
 
             AssertEx.Equal("/optionstrict:custom /out:test.exe test.vb", vbc.GenerateResponseFileContents());
         }
+
+        [ConditionalFact(typeof(UnixLikeOnly)), WorkItem("https://github.com/dotnet/roslyn/issues/80865")]
+        public void SourceFileInRootDirectoryOnUnix()
+        {
+            // On Unix, a source file path starting with "/" without another "/" 
+            // should be prefixed with "./" to avoid being misinterpreted as a command-line switch
+            var vbc = new Vbc
+            {
+                Sources = MSBuildUtil.CreateTaskItems("/Program.vb"),
+            };
+
+            var responseFileContents = vbc.GenerateResponseFileContents();
+            Assert.Contains("/./Program.vb", responseFileContents);
+            Assert.DoesNotContain(" /Program.vb", responseFileContents);
+        }
+
+        [ConditionalFact(typeof(UnixLikeOnly)), WorkItem("https://github.com/dotnet/roslyn/issues/80865")]
+        public void MultipleSourceFilesWithRootDirectoryOnUnix()
+        {
+            // Test multiple files where some are in root and some are not
+            // Also test that /dir/file.vb is NOT transformed (has second '/')
+            var vbc = new Vbc
+            {
+                Sources = MSBuildUtil.CreateTaskItems("/Program.vb", "src/Test.vb", "/App.vb", "/dir/File.vb"),
+            };
+
+            var responseFileContents = vbc.GenerateResponseFileContents();
+            Assert.Contains("/./Program.vb", responseFileContents);
+            Assert.Contains("/./App.vb", responseFileContents);
+            Assert.Contains("src/Test.vb", responseFileContents);
+            // /dir/File.vb should NOT be transformed (contains second '/')
+            Assert.Contains("/dir/File.vb", responseFileContents);
```

Comment:
The other test had a leading space for this assertion, it seems reasonable to have it here also.
```suggestion
            Assert.Contains(" /dir/File.vb", responseFileContents);
```